node/npm/glide/tests/ExportedSymbols.test.ts

```diff
@@ -0,0 +1,191 @@
+import { it } from "@jest/globals";
+import * as f from "fs/promises";
+import { describe } from "node:test";
+import * as ts from "typescript";
+import * as glideApi from "../"; //ESM convention,
+
+describe("Exported Symbols test", () => {
```

Comment:
Maybe reword (up to you)
```suggestion
describe("Validation of Exported Symbols", () => {
```

Reply:
Updated.